Emuelec 4.6 compilation issue FAILED: CMakeFiles/Common.dir/Common/GPU/OpenGL/GLFeatures.cpp.o

### Platform

Linux / BSD

### Compiler and build tool versions

CMAKE 3.24.2

### Operating system version

Ubuntu 20.04 + toolchain

### Build commands used

PROJECT=Amlogic-ce DEVICE=Amlogic-ng ARCH=aarch64 DISTRO=EmuELEC ./scripts/build ppsspp

### What happens

When building 25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7  commit on odroid n2+ emuelec 4.6 the following error pops up

Building CXX object CMakeFiles/Common.dir/Common/GPU/OpenGL/GLFeatures.cpp.o
FAILED: CMakeFiles/Common.dir/Common/GPU/OpenGL/GLFeatures.cpp.o
/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/toolchain/bin/ccache /home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/toolchain/bin/aarch64-libreelec-linux-gnueabi-g++ -DASSETS_DIR=\"/usr/share/ppsspp/assets/\" -DEGL_NO_X11 -DGLEW_NO_GLU -DMINIUPNPC_SET_SOCKET_TIMEOUT -DMINIUPNP_STATICLIB -DPNG_ARM_NEON_OPT=0 -DSHARED_ZLIB -DSTACK_LINE_READER_BUFFER_SIZE=1024 -DUSING_EGL -DUSING_FBDEV -DUSING_GLES2 -DWITH_UPNP -D_BSD_SOURCE -D_DEFAULT_SOURCE -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE=1 -D_POSIX_C_SOURCE=200112L -D_XOPEN_SOURCE=700 -D_XOPEN_SOURCE_EXTENDED -D__BSD_VISIBLE=1 -D__LIBRETRO__ -D__STDC_CONSTANT_MACROS -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/glslang -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/glew -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7 -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/openxr -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/Common -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/libzip -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/libpng17 -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/zstd/lib -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/.aarch64-libreelec-linux-gnueabi -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/snappy/. -I/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/ext/cmake/cpu_features/../../cpu_features/include -march=armv8-a+crc+fp+simd -mabi=lp64 -Wno-psabi -mtune=cortex-a73.cortex-a53 -mno-outline-atomics -Wall -pipe  -O2 -fomit-frame-pointer -DNDEBUG -DDUMMYOPT -Os -D_NDEBUG   -Wformat -Wno-multichar -Wno-psabi -fPIC -fPIC -fno-strict-aliasing -std=gnu++17 -MD -MT CMakeFiles/Common.dir/Common/GPU/OpenGL/GLFeatures.cpp.o -MF CMakeFiles/Common.dir/Common/GPU/OpenGL/GLFeatures.cpp.o.d -o CMakeFiles/Common.dir/Common/GPU/OpenGL/GLFeatures.cpp.o -c /home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/Common/GPU/OpenGL/GLFeatures.cpp
/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/Common/GPU/OpenGL/GLFeatures.cpp: In function 'bool CheckGLExtensions()':
/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/Common/GPU/OpenGL/GLFeatures.cpp:596:30: error: 'GL_COMPRESSED_RGBA_S3TC_DXT5_EXT' was not declared in this scope; did you mean 'GL_COMPRESSED_RGBA_S3TC_DXT1_EXT'?
  596 |                         case GL_COMPRESSED_RGBA_S3TC_DXT5_EXT: gl_extensions.supportsBC123 = true; break;
      |                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                              GL_COMPRESSED_RGBA_S3TC_DXT1_EXT
/home/kelv/EmuELEC/build.EmuELEC-Amlogic-ng.aarch64-4/build/ppsspp-25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7/Common/GPU/OpenGL/GLFeatures.cpp:597:30: error: 'GL_COMPRESSED_RGBA_BPTC_UNORM' was not declared in this scope; did you mean 'GL_COMPRESSED_RGB8_ETC2'?
  597 |                         case GL_COMPRESSED_RGBA_BPTC_UNORM: gl_extensions.supportsBC7 = true; break;
      |                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                              GL_COMPRESSED_RGB8_ETC2

### PPSSPP version affected

25fbfd7a891a7afc36cac4a6ca12bcb59f6411b7 

### Last working version

b44e879d219f96359962f8270f5380cf7eb09a58

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] Make sure to run `git submodule update --init --recursive` before building.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try deleting the build directory and running the build again.
- [X] Check GitHub Actions, which builds every merge and PR.
- [X] Include logs and help us reproduce.

--- a/Common/GPU/OpenGL/DataFormatGL.cpp
+++ b/Common/GPU/OpenGL/DataFormatGL.cpp
@@ -86,7 +86,7 @@ bool Thin3DFormatToGLFormatAndType(DataFormat fmt, GLuint &internalFormat, GLuin
 		alignment = 16;
 		break;
 
-#if PPSSPP_PLATFORM(WINDOWS)
+#ifndef USING_GLES2
 	case DataFormat::BC1_RGBA_UNORM_BLOCK:
 		internalFormat = GL_COMPRESSED_RGB_S3TC_DXT1_EXT;
 		format = GL_RGB;

--- a/Common/GPU/OpenGL/GLFeatures.cpp
+++ b/Common/GPU/OpenGL/GLFeatures.cpp
@@ -593,10 +593,8 @@ bool CheckGLExtensions() {
 			switch (compressedFormats[i]) {
 			case GL_COMPRESSED_RGB8_ETC2: gl_extensions.supportsETC2 = true; break;
 			case GL_COMPRESSED_RGBA_ASTC_4x4_KHR: gl_extensions.supportsASTC = true; break;
-#if !PPSSPP_PLATFORM(IOS) && !PPSSPP_PLATFORM(MAC)
+#ifndef USING_GLES2
 			case GL_COMPRESSED_RGBA_S3TC_DXT5_EXT: gl_extensions.supportsBC123 = true; break;
-#endif
-#if PPSSPP_PLATFORM(WINDOWS)
 			case GL_COMPRESSED_RGBA_BPTC_UNORM: gl_extensions.supportsBC7 = true; break;
 #endif
 			}

